src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -672,6 +672,9 @@ public void instantiateDemoUsers() {
       demoUser1.setName("Demo User 1");
       demoUser1.setGender(Gender.MALE);
       demoUser1.setBirthday(new Date());
+      demoUser1.setMaxAge(30);
+      demoUser1.setMinAge(18);
```

Comment:
Is 18 by default..

Reply:
@serainaschraff 
![image](https://user-images.githubusercontent.com/75019468/169706740-c6c158c9-f8f1-4eea-8de7-87ab41afdc8f.png)
@TwoDigitsOneNumber You can fix that by making the users older than 18 years old, because we allow only for users that are older than 18!
@JoMuelle Can you fix findMaxAgeDefault() to return at least 18? + throw error in createUser if user.birthday is not > years ago?

Reply:
@daveking7 that is not a responsibility of this PR/branch. as you said, this should be addressed in the createUser function, then it is also covered with the guard, as this checks for a user, but that younger user should never enter the app in the first place

Reply:
@TwoDigitsOneNumber But why did you add those preferences then? ;)